templates/index.html

```diff
@@ -38,184 +43,263 @@ <h1 dir="ltr">IATI Data Store CSV Query Builder (Alpha Version)</h1>
 
         <form action="index.php" method="post" target="_self" onsubmit="">
 
-          <div class="form-row">
-            <div class="col">
-              <label for="entry_1689841214">
-                <div>Choose format
-                  <label for="itemView.getDomIdToLabel()" aria-label="(Required field)"></label>
-                </div>
-              </label>
-              <ul>
-                <li>
-                  <label>
-                    <input type="radio" name="entry.1085079344" value="activity" id="group_1085079344_1" aria-label="Activity"
-                    {% if not dataset or (dataset and dataset == "activity") %}
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>One activity per row</span>
-                  </label>
-                </li>
-                <li>
-                  <label>
-                    <input type="radio" name="entry.1085079344" value="transaction" id="group_1085079344_2" aria-label="Transaction"
-                    {% if dataset and dataset == "transaction" %}
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>One transaction per row</span>
-                  </label>
-                </li>
-                <li>
-                  <label>
-                    <input type="radio" name="entry.1085079344" value="budget" id="group_1085079344_3" aria-label="Budget"
-                    {% if dataset and dataset == "budget" %}
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>One budget per row</span>
-                  </label>
-                </li>
-              </ul>
+
+          <h2>What information are you looking for?</h2>
+          <p>These options let you filter the information you are looking for. Additional filters <a href="http://datastore.iatistandard.org/docs/api/#filtering">are available</a> by querying the Datastore manually.</p>
+
+          <div class="step-section">
+            <div class="title">
+              <span class="step">1</span>
+              <h3>Select who reports</h3>
```

Comment:
Suggest changing `reports` to something else - it doesn't seem like quite the right term.

(unhelpfully, I can't immediately think of something better)

Reply:
I've gone for `Select the data source` for the time being...